qabelbox/src/main/java/de/qabel/qabelbox/services/QabelService.java

```diff
@@ -192,23 +104,11 @@ public void onCreate() {
 
         incomingHandlerThread.start();
 
-        contacts = new HashMap<>();
-        identities = new HashMap<>();
-
         mNotificationManager = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
 
         setNotification("Qabel Service waiting for database unlock");
```

Comment:
This notification text could be changed, since there is no waiting for a database unlock any more.